pew/pew.py

```diff
@@ -81,6 +82,42 @@ def get_project_dir(env):
 
     return project_dir
 
+#class to support reading ini file
```

Comment:
This might be better suited and expanded as a docstring for the `EnvParser` class.